examples/cityscapes/README.md

```diff
@@ -0,0 +1,182 @@
+
+```
+include:
+    - leap_binder.py
+    - utils_all/gcs_utils.py
+    - utils_all/general_utils.py
+    - utils_all/metrics.py
+    - utils_all/preprocessing.py
+    - visualizers/visualizers.py
+    - yolo_helpers/yolo_utils.py
+    - project_config.py
+```
+
+### **leap_binder.py file**
+`leap_binder.py` configure all binding functions used to bind to Tensorleap engine. These are the functions used to 
+evaluate and train the model, visualize the variables, and enrich the analysis with external metadata variables
+
+### Testing
+
+To test the system we can run `leap_test.py` file using poetry:
+
+```
+poetry run test
+```
+
+This file will execute several tests on [the leap_binder.py] script to assert that the implemented binding functions: 
+preprocess, encoders,  metadata, etc,  run smoothly.
+
+*For further explanation please refer to the [docs](https://docs.tensorleap.ai/)*
+
+## Project Description
+_____________________________________________
+### YOLOv7 model with Cityscapes dataset
+
+This project utilizes the [YOLOv7]() algorithm to perform object detection on [Cityscapes]() dataset. The 
+cityscapes dataset is a large-scale dataset that contains a diverse set of stereo video sequences recorded in street 
+scenes from 50 different cities, with high quality pixel-level annotations of 
+5 000 frames in addition to a larger set of 20 000 weakly annotated frames.
+
+### Latent Space Exploration
+The following plot illustrates a population exploration map, depicting the similarity among samples based on the latent 
+space of a trained model. This map is constructed using the extracted features of the model.
+A distinct separation is evident between images with high and low minimum values (min). This suggests that images with
+high min share common features that differ from those found in images with low min.
+
+![input_min](screen_shots/input_min.png)
+
+
+#### *Detecting & Handling High Loss Clusters*
+Upon conducting additional analysis, it was found that when the number of small bounding boxes decreases, 
+the object detection loss increases.
+
+![high_loss_small_bboxes](screen_shots/high_loss_small_bboxes.png)
+
+Furthermore, When the number of labeled persons decreases, the object detection loss increases. But not necessarily the 
+other direction is also true-There can be a variety of reasons why the loss is high.
+
+![high_loss_number_of_persons](screen_shots/high_loss_number_of_persons.png)
+
+By filtering the dot size in the latent space based on the image mean, we observe a cluster of samples that exhibit a 
+higher mean. The proximity of these samples suggests that they share similar features, indicating a potential 
+correlation between their characteristics.
+
+![high_loss_mean_image](screen_shots/high_loss_mean_image.png)
+
+#### *Mislabeled data*
+
+Upon conducting further analysis, it has been discovered that images containing cars exhibit a high loss. Subsequent 
```

Comment:
This is cool! are we sure this is the case? saying CityScapes have an issue with car labelling (more than other objects) is not trivial.

Reply:
There is mislabeling in general- happed in persons and building also but in cars it is definitely the most common one
<img width="678" alt="Screenshot 2023-08-08 at 14 31 03" src="https://github.com/tensorleap/tensorleap/assets/135505576/90e5aed1-b4ff-4db3-b643-341b984f574c">